modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ExtensionMethods.cs

```diff
@@ -165,6 +166,35 @@ public static string NameWithTypeParameters(this INamedTypeSymbol symbol)
         public static string FullQualifiedName(this INamespaceSymbol namespaceSymbol)
             => namespaceSymbol.ToDisplayString(FullyQualifiedFormatOmitGlobal);
 
+        public static string FullQualifiedName(this ISymbol symbol)
+            => symbol.ToDisplayString(FullyQualifiedFormatOmitGlobal);
+
+        public static string FullQualifiedSyntax(this SyntaxNode node, SemanticModel sm, bool isFirstNode = true)
+        {
+            if (node is NameSyntax ns && isFirstNode)
+            {
+                SymbolInfo nameInfo = sm.GetSymbolInfo(ns);
+                return nameInfo.Symbol?.FullQualifiedName() ?? ns.ToString();
+            }
+
+            StringBuilder sb = new StringBuilder(128);
```

Comment:
Why use a capacity of 128?

Reply:
It's just an hint, but 128 is probably longer than we'd usually want. Would you prefer it at something smaller like 64 or would you rather we just not provide a size hint at all?

Reply:
It's not just a hint, the capacity determines the size of the internal array that will be allocated so I'd rather not allocate a bigger array than we need. It's probably better to use the parameterless constructor so it uses the default capacity, I'm sure the default value is likely to work best in most situations which is why it was chosen.

Reply:
Done.